README.adoc

```diff
@@ -116,9 +129,26 @@ base:jvm_uptime_seconds 39.025
 ----
 [role="no_copy"]
 ----
```

Comment:
The base metrics here should be `base_*` not `base:*`

Reply:
Fixed.